### PR TITLE
Redirect t10n [TA-49]

### PR DESCRIPTION
**Pull request by ipeluffo**
https://gr4vy.atlassian.net/browse/TA-49

### CODE REVIEW

File: versioned_docs/version-1.1.0/payment-methods/statuses.mdx [hunk 1]
```diff
@@ -8,39 +8,26 @@ import Mermaid from "@theme/Mermaid";
...
+    processing --> buyer_approval_required
     processing --> stored
-    processing --> used
-    buyer_approval_pending --> buyer_approval_declined
-    buyer_approval_pending --> buyer_approval_declined
-    buyer_approval_pending --> buyer_approval_timedout
-    buyer_approval_pending --> stored
-    buyer_approval_pending --> used
+    processing --> failed
+    buyer_approval_required --> stored
```
Comment (cbetta) [inline versioned_docs/version-1.1.0/payment-methods/statuses.mdx:30]: `stored` is not in the list of statuses above but it is mentioned here.

**Review by cbetta — CHANGES_REQUESTED** [review]
Sorry about this, the changes look good but they're in the wrong version.

Updates should not happen to any of the "versioned" docs, instead they should be on the root folder. That way, when we are happy we can tag all these changes as a new release.

😬

File: versioned_docs/version-1.1.0/payment-methods/statuses.mdx (hunk 1 above)
Comment (ipeluffo) [reply versioned_docs/version-1.1.0/payment-methods/statuses.mdx]: Fixed in https://github.com/gr4vy/gr4vy-docs/pull/47/commits/47e3827b556acc25a867dfdc3ea005ff410d3a3b

**Comment by ipeluffo** [comment]
> Sorry about this, the changes look good but they're in the wrong version.
> 
> Updates should not happen to any of the "versioned" docs, instead they should be on the root folder. That way, when we are happy we can tag all these changes as a new release.
> 
> 😬

Sorry, my bad... I've fixed this, hopefully it should correct now

**Review by cbetta — APPROVED** [review]

**Comment by cbetta** [comment]
Great work. :shipit:

**Pull request merged**